exercises/practice/simple-cipher/.meta/src/SimpleCipher.example.elm

```diff
@@ -0,0 +1,58 @@
+module SimpleCipher exposing (decode, encode, keyGen)
+
+import Random exposing (Generator)
+
+
+type Direction
+    = Encode
+    | Decode
+
+
+encode : String -> String -> String
+encode =
+    transform Encode
+
+
+decode : String -> String -> String
+decode =
+    transform Decode
+
+
+transform : Direction -> String -> String -> String
```

Comment:
I would probably separate this in to two functions (encode and decode) and have helper functions for the shared bits. It's a minor point though and I wouldn't be offended if you disagree.

Probably something like this

```
decode : String -> String -> String
decode = key text
    List.map2 decodeCombine (String.toList text) (longKey text)
        |> String.fromList
```

Reply:
Well, I already had encode and decode, and this was the helper function, but I made some changes in [b6e57ae](https://github.com/exercism/elm/pull/619/commits/b6e57ae20cc945af2fd932f712989657d39a11a2), it it more clear?

Reply:
Yep, that looks good. Maybe encodeHelper is the wrong name though, and the original transform name is better?